recipes/spirv-tools/all/patches/support-SPV_KHR_fragment_shading_rate.patch

```diff
@@ -0,0 +1,220 @@
+Patch from https://github.com/KhronosGroup/SPIRV-Tools/commit/a1d38174b1f7d2651c718ae661886d606cb50a32
```

Comment:
I'm ok with the patch.
Looking at the error, some enum value was used without definition.
Looks like an error to me.

Reply:
those enums comes from spirv-headers. But spirv-tools sticks to specific commits of spirv-headers, not to tags, so it's hard to package this ecosystem for package managers.